runner/ansible/roles/2.1.5/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.1.5'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    #  if azure-lb RA is not configured, then fail
+    cibadmin -Q --xpath "//primitive[@type='azure-lb']/@type" || exit 1
+    # if not grouped, exit with fail
+    [[ $(cibadmin -Q --xpath "//group/primitive[@type='azure-lb']/@type") != "0" ]] && exit 1
```

Comment:
We need to change this call. I would do the same that you did in the 2.1.4 check.

```
cibadmin -Q --xpath "//group/primitive[@type='azure-lb']/@type" || exit 1
```

Reply:
done